src/framework/theme/components/route-tabset/route-tabset.component.ts

```diff
@@ -63,14 +63,15 @@ import { convertToBoolProperty } from '../helpers';
   selector: 'nb-route-tabset',
   styleUrls: ['./route-tabset.component.scss'],
   template: `
-    <ul>
+    <ul class="route-tabs">
       <li *ngFor="let tab of tabs"
           (click)="$event.preventDefault(); selectTab(tab)"
           routerLink="{{tab.route}}"
           routerLinkActive="active"
           [routerLinkActiveOptions]="{ exact: true }"
-          [class.responsive]="tab.responsive">
-        <a href>
+          [class.responsive]="tab.responsive"
+          class="route-tab">
+        <a href (click)="$event.preventDefault()">
```

Comment:
Why do we need `preventDefault` here?
It it's required, let's move it in the separate PR